cypress/integration/newsBodySpec.js

```diff
@@ -47,8 +47,26 @@ describe('Article Body Tests', () => {
     visibleImageNoCaption(getElement('figure').eq(0));
   });
 
-  it('should have a visible image with a caption', () => {
-    visibleImageWithCaption(getElement('figure').eq(2));
+  it('should have a visible image with a caption that is lazyloaded and has a noscript fallback image', () => {
+    let thirdFigure;
```

Comment:
Why third?

Reply:
Because we can guarantee this is below the fold, which means that we can 
1 - more reliably test that the image has not loaded when the page loads initially, and instead has an empty placeholder
2 - more reliably test that when scrolling the page, the empty placeholder is replaced with the actual image